src/e2e/features/3-open-formulieren.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Resident submits form in open-forms
+
+  Scenario: Resident fills in indienen-aansprakelijkheid-behandelen zaaktype
+    Given A Resident fills in the indienen-aansprakelijkheid-behandelen open-forms form using user profile "alice"
+    When "Alice" clicks on element with text: "Verzenden"
+    Then "Bob" navigates to "zac" with path "/zaken/werkvoorraad"
```

Comment:
This is not a resulting state, Bob still has to navigate and do things. Any actions should be executed in the "When" block.

Also, we should use business speak in these scenarios, so without technical terms like element, navigate or path...

I would write this scenario as something like:
```gherkin
Scenario: Resident fills in indienen-aansprakelijkheid-behandelen zaaktype
    Given Resident "Alice" fills in the indienen-aansprakelijkheid-behandelen open-forms form
    When "Alice" submits the open-forms form
    And Employee "Bob" is working in ZAC
    And "Bob" opens the "werkvoorraad"
    Then "Bob" can see the case that "Alice" has created through open-forms
```

Reply:
Well this causes some code duplication, so i will try the best i can to change it to this format without code duplication

Reply:
Yes, all good 👍 
We should ensure we define these steps in a common way, and use the business language to describe them. I approve now, but think we need to get the business testers / users to agree with the terminology and descriptions used.